views/styles/chart_styles.js

```diff
@@ -147,6 +147,9 @@ export const chartStyles = (theme) => ({
   viewChartRow: {
     display: 'flex',
     justifyContent: 'space-between',
+    inlineSize: '800px',
+    overflowWrap: 'break-word',
+    paddingBottom: '50px',
```

Comment:
do we need these style rules? I would think the removal of the `pre` tag would be enough.

Reply:
👍 
https://github.com/AMP-SCZ/dpdash/pull/270/commits/c603f0a36c4c4f2e2f57023655f2099b7f9c469b